### PR TITLE
docs(PPDSC-2010): Transitions presets

### PR DESCRIPTION
**Pull request by Srinivasan-Ramasamy**
PPDSC-2010

**What**

1. Updated transitions presets doc page.
2. Created transitions videos.

**I have done:**
 - [ ] Written unit tests against changes
 - [ ] Written accessibility tests against the component
 - [ ] Written functional tests against the component and/or NewsKit site
 - [ ] Updated relevant documentation
 - [ ] Updated the release notes for the [next release version](https://nidigitalsolutions.jira.com/wiki/spaces/NPP/pages/2349695021/Release%2BNotes%2BCode%2B-%2BWeb%2BDocumentation)
 - [ ] Discussed with QA and/or design about any visual changes

**I have tested manually:**
 - [ ] Renders correctly on IE11
 - [ ] The feature's functionality is working as expected on other browsers such as Safari, Firefox and IE11
 - [ ] There are no accessibility errors reported by WAVE
 - [ ] The screen reader reads the expected text for the tabbable elements.
 - [ ] There are no new errors in the browser console coming from this PR.
 - [ ] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
 - [ ] The Playground feature is working as expected

**Before**

**After:**

**Who should review this PR**

**How to test**

### CODE REVIEW

**Comment by newskit-engineering** [comment]
You can preview these changes on: 
  Documentation Site: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2010-transitions-presets-page 
  Storybook: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2010-transitions-presets-page/storybook

File: site/pages/theme/presets/transition-presets.tsx [hunk 1]
```diff
@@ -0,0 +1,1032 @@
...
+    tokenName: 'transitionDelay',
+    acceptedValues: <Link href="/theme/motion/#duration">motionDuration</Link>,
+    example: <Mono>motionDuration000</Mono>,
+    description:
+      'Specifies how many milliseconds delay for the transition effect to start',
+  },
+  {
+    tokenName: 'transitionDuration',
+    acceptedValues: <Link href="/theme/motion/#duration">motionDuration</Link>,
+    example: <Mono>motionDuration030</Mono>,
```
Comment (baburay23) [inline site/pages/theme/presets/transition-presets.tsx:47]: hey, looks good but i think the transitionDuration is incorrect. The default in the code base is `motionDuration020`

**Review by mutebg — COMMENTED** [review]
Looks good, 1 question only

**Comment by Srinivasan-Ramasamy** [comment]
Thanks @baburay23 and @mutebg for reviewing the code :+1:

File: site/pages/theme/presets/transition-presets.tsx [hunk 2]
```diff
@@ -0,0 +1,1032 @@
...
+const VideoElement = ({url}: {url: string}) => (
+  <video
+    autoPlay
+    controls
+    controlsList="nofullscreen nodownload"
+    loop
+    src={url}
+    width="100%"
+  >
+    <track kind="captions" />
```
[the remark this answers is not in the document]
Comment (Srinivasan-Ramasamy) [reply site/pages/theme/presets/transition-presets.tsx:400]: Yes @mutebg, 

We're getting - a11y error:  Media elements such as <audio> and <video> must have a <track> for captions.

File: site/pages/theme/presets/transition-presets.tsx (hunk 1 above)
Comment (Srinivasan-Ramasamy) [reply site/pages/theme/presets/transition-presets.tsx]: Sure @baburay23, I've corrected with motionDuration020. Sorry I was reffered confluence documentation page.

File: site/pages/theme/presets/transition-presets.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (mutebg) [reply site/pages/theme/presets/transition-presets.tsx:400]: thanks, I didn't think about it.

**Review by mutebg — APPROVED** [review]

**Review by baburay23 — APPROVED** [review]

**Pull request merged**